grizzly_extras/types.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum, auto
+
+
+class ResponseContentType(Enum):
```

Comment:
To get rid of `grizzly_extras.types`, this should be moved into `transformer.py` and renamed to `TransformerContentType`.

Also, `str_response_content_type` should be added as method in the enum, something like `from_string` and:
```
@classmethod
def from_string(value: str) -> 'TransformerContentType':
  ...if cases...
```